pkg/networkservice/common/excludedprefixes/server.go

```diff
@@ -44,7 +45,7 @@ type excludedPrefixesServer struct {
 }
 
 func (eps *excludedPrefixesServer) init() {
-	logger := trace.Log(eps.ctx)
```

Comment:
For initialisation purpose please use direct logger from tools instead of taking if from context.